src/view/OutputView.js

```diff
@@ -0,0 +1,35 @@
+import MESSAGE from '../constant/Message.js';
+
+const OutputView = {
+  print(message) {
+    console.log(message);
+  },
+  printPurchaseCount(count) {
+    this.print(`${count}개를 구매했습니다.`);
+  },
+  printLotto(LottoNumbers) {
+    this.print(
+      '[' + LottoNumbers.map(number => number.toString()).join(', ') + ']',
+    );
+  },
+  printWinningCharacteristic(winningResult) {
+    this.print('');
+    this.print(MESSAGE.winningCharacteristicsHeader);
+    this.print(MESSAGE.lineSplitter);
+    this.print(
+      Object.entries(winningResult)
+        .sort((a, b) => a[0].localeCompare(b[0]))
```

Comment:
```suggestion
       .sort((a, b) => Number(a[0]) - Number(b[0]))
```
nit) 숫자 정렬이라면 이런 방법도 있습니다

nit은 참고차 혹은 꼭 반영하지 않아도 되는 내용입니다~